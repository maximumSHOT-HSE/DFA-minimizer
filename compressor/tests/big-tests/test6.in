12 72 6
1 1 0
1 6 1
1 10 2
1 3 3
1 4 4
1 3 5
2 6 0
2 6 1
2 6 2
2 6 3
2 10 4
2 6 5
3 4 0
3 3 1
3 10 2
3 3 3
3 9 4
3 8 5
4 12 0
4 6 1
4 4 2
4 8 3
4 6 4
4 7 5
5 5 0
5 8 1
5 6 2
5 8 3
5 1 4
5 2 5
6 5 0
6 4 1
6 6 2
6 4 3
6 5 4
6 5 5
7 4 0
7 1 1
7 3 2
7 10 3
7 11 4
7 1 5
8 2 0
8 2 1
8 6 2
8 1 3
8 2 4
8 7 5
9 10 0
9 12 1
9 5 2
9 4 3
9 5 4
9 4 5
10 10 0
10 3 1
10 7 2
10 9 3
10 5 4
10 11 5
11 4 0
11 4 1
11 12 2
11 11 3
11 12 4
11 10 5
12 9 0
12 6 1
12 10 2
12 7 3
12 5 4
12 5 5
5
2
7 9 
