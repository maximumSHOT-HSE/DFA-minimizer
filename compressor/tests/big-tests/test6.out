12 72 6
1 1 0
7 1 1
8 1 3
5 1 4
7 1 5
8 2 0
8 2 1
8 2 4
5 2 5
3 3 1
10 3 1
7 3 2
1 3 3
3 3 3
1 3 5
3 4 0
7 4 0
11 4 0
6 4 1
11 4 1
4 4 2
6 4 3
9 4 3
1 4 4
9 4 5
5 5 0
6 5 0
9 5 2
6 5 4
9 5 4
10 5 4
12 5 4
6 5 5
12 5 5
2 6 0
1 6 1
2 6 1
4 6 1
12 6 1
2 6 2
5 6 2
6 6 2
8 6 2
2 6 3
4 6 4
2 6 5
10 7 2
12 7 3
4 7 5
8 7 5
5 8 1
4 8 3
5 8 3
3 8 5
12 9 0
10 9 3
3 9 4
9 10 0
10 10 0
1 10 2
3 10 2
12 10 2
7 10 3
2 10 4
11 10 5
11 11 3
7 11 4
10 11 5
4 12 0
9 12 1
11 12 2
11 12 4
5
2
7 9 
